deployment/pypi/README_zh_CN.md

```diff
@@ -1,42 +1,72 @@
 # 用于 NNI 的 python 包索引 (pypi)
 
-## 1. 说明
+这是用于 NNI 项目 PyPI 生成和上传的工具。
 
-这是用于 NNI 项目的 PyPI 生成和上传的工具。
+## **Linux**
 
-## 2.准备环境
+* __准备环境__
 
-在生成和上传 NNI 包之前，确保使用了下列环境。
+  在生成和上传 NNI 包之前，确保使用了下列环境。
```

Comment:
Please do not modify the Chinese document directly here. NNI unifies the Chinese corresponding document in another place.